pkg/sippyserver/dbloader.go

```diff
@@ -411,6 +416,51 @@ func appendJobIssuesFromVariants(jobCache map[string]*models.ProwJob, jobIssues
 	return nil
 }
 
+const watchlistLabel = "sippy-watchlist"
+
+// TestIsOnWatchlist returns true if the test matches one of our hardcoded regexes, or
+// has an associated bug with the sippy-watchlist label.
+func TestIsOnWatchlist(test *models.Test) bool {
+	watchlistREs := []*regexp.Regexp{
+		regexp.MustCompile("events should not repeat pathologically$"),
+		regexp.MustCompile("Check if alerts are firing during or after upgrade success"),
+		regexp.MustCompile("Alerts shouldn't report any unexpected alerts in firing or pending state"),
+	}
+	for _, re := range watchlistREs {
+		if re.MatchString(test.Name) {
+			return true
+		}
+
+		for _, bug := range test.Bugs {
+			if util.StrSliceContains(bug.Labels, watchlistLabel) {
+				return true
+			}
+		}
+	}
+	return false
+}
```

Comment:
This would be better in `pkg/testidentification`, and having the regexes outside the function would be more efficient.  These get recompiled on each function call.

Reply:
Another reason to have these outside the function call like this [1] is because you'll always compile them when sippy starts, meaning you WILL get a panic if you mess up the regex.  Having them  in a function means you won't know until the function is actually called.

[1] https://github.com/openshift/sippy/blob/master/pkg/testidentification/test_identification.go#L163-L173